source/common/tracing/zipkin/span_context.cc

```diff
@@ -75,22 +75,11 @@ SpanContext::SpanContext(const Span& span) {
   id_ = span.id();
   parent_id_ = span.isSetParentId() ? span.parentId() : 0;
 
-  for (const Annotation& annotation : span.annotations()) {
```

Comment:
You are removing the annotations from the Zipkin span context. Are you still propagating the `x-ot-span-context` header? Even though this header should eventually go away for the Zipkin-related code as part of issue #1364, I am wondering how you are currently handling it. It is not entirely clear to me if you have changed all parts of the Zipkin code related to that. Could you, please, comment on that?

Reply:
I am not removing the annotations themselves - they are still associated with the spans. The only thing that has been removed is the flags that indicate whether a particular annotation was set of the previous context.

The `x-ot-span-context` header is still being propagated as before - just without the additional annotation flags. This should actually make it easier to do #1364, as otherwise you would have to find another way to transfer these annotation flags, as the standard B3 headers don't carry that information.

Reply:
Indeed.